templates/welcome_email.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+
+</head>
+
+<body>
+    <h3>Hi {{firstName}},</h3>
+    <p>Thanks for filling out the form on <a href="https://www.rememberthee.com" target="_blank">rememberthee.com</a>.
+        Remember Thee is available for download on iOS only. <br>You will receive an email from Apple's app testing
```

Comment:
Don't think we need the `<br>`, why did you include it?

Reply:
The <p> was on one line and it was too long. Do you want to earise?

Reply:
Yes, get rid of the `<br>`, you see how the text wraps around on its own once I get rid of it from the code?
<img width="915" alt="Screen Shot 2021-11-18 at 10 18 30 AM" src="https://user-images.githubusercontent.com/5726560/142473746-99d15af2-a0df-43ce-ad07-376f98b91fa3.png">